nav2_costmap_2d/test/integration/inflation_tests.cpp

```diff
@@ -137,7 +138,8 @@ void TestNode::validatePointInflation(
         double dist = std::hypot(dx, dy);
 
         unsigned char expected_cost = ilayer->computeCost(dist);
-        ASSERT_TRUE(costmap->getCost(cell.x_, cell.y_) >= expected_cost);
+//        ASSERT_TRUE(costmap->getCost(cell.x_, cell.y_) >= expected_cost)
```

Comment:
Why is this commented out?

Reply:
Was testing out something,, mistakenly commented out.
Fixed.